packages/web-app-files/src/components/AppBar/AppBar.vue

```diff
@@ -73,7 +73,8 @@ export default {
     hasBulkActions: { type: Boolean, default: false },
     hasSharesNavigation: { type: Boolean, default: false },
     hasSidebarToggle: { type: Boolean, default: true },
-    hasViewOptions: { type: Boolean, default: true }
+    hasViewOptions: { type: Boolean, default: true },
+    showActionsOnSelection: { type: Boolean, default: false }
```

Comment:
wouldn't setting `hasBulkActions` to `false` achieve the same result? 🤔

Reply:
hm... no, it wouldn't. but should it? 🤔

Reply:
Not quite, as "Create Space" is not a bulk action. The button would still disappear. Currently, we don't have bulk actions for spaces because you can only select on space at a time.

Reply:
The thing is, it will still show the selection info which makes the create space button disappear.

Reply:
Yeah you're right... I mean, we currently only need this for spaces. They are kind of a "special" case because they don't have multiselect and a selected space is not visibly selected for the user (other than in the sidebar). Therefore `showActionsOnSelection` would be the easiest fix without touching the rest.

Reply:
You're right, that's probably the most generic we can get and (semantically) merging with other props doesn't make too much sense. Thanks for discussing it 😃